src/components/OptionRow.js

```diff
@@ -266,6 +271,11 @@ const OptionRow = (props) => {
                                         <Icon src={Expensicons.Pin} height={16} width={16} />
                                     </View>
                                 )}
+                                {Boolean(props.option.customIcon) && (
+                                    <View>
+                                        <Icon src={props.option.customIcon.src} height={16} width={16} fill={props.option.customIcon.color} />
+                                    </View>
+                                )}
```

Comment:
Note: I think we can use this generic option prop to replace `hasDraftComment` & `hasOutstandingIOU` above, maybe in a separate PR